lib/tessel/deploy.js

```diff
@@ -205,49 +211,98 @@ actions.sendBundle = function(tessel, filepath, opts) {
 
 actions.tarBundle = function(opts) {
   var target = opts.target || process.cwd();
+  var pack = tar.Pack();
+  var buffers = [];
+
+  if (opts.slim) {
+    console.log('Running slim...');
```

Comment:
@HipsterBrown if you feel output is necessary, can you use `logs.basic` for consistency with the rest of the library? (Other options are `logs.warn` and `logs.err`)